src/Tools/CoreFx.Tools/GenerateConfigurationProps.cs

```diff
@@ -101,6 +101,20 @@ private void ParseProperties(string propertyName, ProjectRootElement project, bo
                     AddProperties(whenPropertiesElement, value, includeAdditionalProperites, parsedValuePrefix);
                 }
 
+                if (property.Insignificant)
+                {
+                    // for insignificant properties, don't overwrite with default if already set, but derive values from it.
```

Comment:
No because if there happened to be derived property values those wouldn't be set.